docs/4.x/tagmanager/javascript-api-reference.md

```diff
@@ -66,7 +66,11 @@ enabled. It will not throw an error. An error will be only thrown if `TagManager
 * `TagManager.dom.getDocumentWidth()` - Get the full width of the entire webpage in pixels.
 * `TagManager.dom.addEventListener(element, eventType, eventHandler, useCapture = false)` - Attach an event listener for the given event type to the given element and executes the event handler once the event occurs.
 * `TagManager.dom.getElementClassNames(element)` - Get a string containing all class names this element has defined. Please note that multiple whitespaces will be removed and each class name will be separated by one whitespace.
-* `TagManager.dom.getElementAttribute(element, attributeName)` - Get the attribute of an element. Returns `null` if the attribute does not exist, an empty string if the attribute exists but has no value, or the value of the attribute.
+* `TagManager.dom.getElementText(element)` - Get a string containing the text of the element. Returns `null` if the element doesn't exist. All whitespace character groups are replaced with a single space. Since Matomo 4.12, this method will mask the value if the `data-matomo-mask` attribute is set. 
+* `TagManager.dom.getElementAttribute(element, attributeName)` - Get the attribute of an element. Returns `null` if the attribute does not exist, an empty string if the attribute exists but has no value, or the value of the attribute. Since Matomo 4.12, this method will mask the value of certain attributes if the `data-matomo-mask` attribute is set.
+* `TagManager.dom.shouldElementBeMasked(element)` - Returns whether or not the specified element should be masked. If the element doesn't exist, `false` will be returned.
```

Comment:
```suggestion
* `TagManager.dom.shouldElementBeMasked(element)` - Returns whether or not the specified element should be masked. If the element doesn't exist, `false` will be returned(Available for Matomo 4.12 and above).
```

Reply:
We should also say it returns a boolean value, so its clear that it will return true/false